src/applications/hca/components/IDForm.jsx

```diff
@@ -82,6 +84,52 @@ export default class IDForm extends React.Component {
       'ui:labels': genderLabels,
     },
   };
+
+  renderContinueButtonOrStatus = () => {
+    const { enrollmentStatus } = this.props;
+    if (enrollmentStatus && enrollmentStatus !== 'none_of_the_above') {
+      return (
+        <React.Fragment>
+          <AlertBox
+            isVisible
+            status="error"
+            headline="Please sign in to continue your application"
+            content={
+              <React.Fragment>
+                <p>
+                  We’re sorry for the interruption, but we need you to review
+                  some information before you continue applying. Please sign in
+                  below to review. If you don’t have an account, you can create
+                  one now.
+                </p>
+                <button
+                  className="usa-button-primary"
+                  onClick={this.props.handleSignIn}
+                >
+                  Sign in to VA.gov
+                </button>
+              </React.Fragment>
+            }
+          />
+          <br />
+        </React.Fragment>
+      );
```

Comment:
This fragment was originally one level above in the `IDPage` container. I moved it here since I noticed in mocks that we are showing either the submit button or the prompt at the bottom (never both).